components/solve_crm/sources/new-company/new-company.mjs

```diff
@@ -0,0 +1,38 @@
+import common from "../common/base.mjs";
+
+export default {
+  ...common,
+  key: "solve_crm-new-company",
+  name: "New Company (Instant)",
```

Comment:
We should update all the source keys in accordance with [the updated guidelines](https://pipedream.com/docs/components/guidelines/#component-key-pattern):
`Company Created (Instant)`
or
`New Company Created (Instant)`